lib/request/middleware/index.js

```diff
@@ -3,13 +3,9 @@
 // Allows incoming requests to be pre-processed, transformed, or sent elsewhere
 'use strict';
 
-var apiTree, existslib, pathlib;
-
-pathlib = require('path');
-
-existslib = process.version.split('.')[1] === '6' && require('path') || require('fs');
-
-apiTree = require('apitree');
+var pathlib = require('path');
+var fs = require('fs');
+var apiTree = require('apitree');
```

Comment:
Yes I added it, but didn't look at configuration. Sry. I'll try to fix it, it's a bit noisy atm.

This might fix it https://github.com/socketstream/socketstream/commit/c14129f172b940fda7d908aa94fe463e6039f6cc